docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -456,8 +456,9 @@ relation to port exhaustion.
 
 ## Indexer (`indexer`)
 
-- `ISSUE_INDEXER_TYPE`: **bleve**: Issue indexer type, currently supported: `bleve`, `db` or `elasticsearch`.
-- `ISSUE_INDEXER_CONN_STR`: ****: Issue indexer connection string, available when ISSUE_INDEXER_TYPE is elasticsearch. i.e. http://elastic:changeme@localhost:9200
+- `ISSUE_INDEXER_TYPE`: **bleve**: Issue indexer type, currently supported: `bleve`, `db`, `elasticsearch` or `meilisearch`.
+- `ISSUE_INDEXER_CONN_STR`: ****: Issue indexer connection string, available when ISSUE_INDEXER_TYPE is elasticsearch, or meilisearch. i.e. http://elastic:changeme@localhost:9200
+- `ISSUE_INDEXER_CONN_AUTH`: ****: Issue indexer Authentication token, available when ISSUE_INDEXER_TYPE is meilisearch
```

Comment:
Could we put the token into the conn_str?

Reply:
We could, but I went with a separate option because asking users to put their conn_string as `http://:PASSWORD@127.0.0.1:7700/` would be slightly confusing as we'd need to specific to always put a `:` in front of the password, and also properly escape it.

I am ok with either way if anyone feels strongly about a certain direction.

Reply:
It's better to keep it out of URL

Reply:
keeping it in URL keeps it consistent with elasticsearch